localisation/00_zuwc_united_localisation.csv

```diff
@@ -2789,21 +2778,21 @@ mind_control_2;Steady Mind Control;;Stabile Gedankenkontrolle;;;;;;;;;;;x
+pet_felguard_desc;This character has a felguard minion. Felguard are demons of the mo'arg race that serve as the Legion's rank and file demonic soldiers and front-line troops. They are fearless and unwaveringly loyal to their master's godlike will. They are almost always seen with heavy, two-handed weaponry. They are slow to strike, but hit quite hard, and are more agile than they look.;;Dieser Charakter hat eine Teufelswache als Lakaien. Teufelswachen sind D�monen der Mo'arg-Rasse, die als d�monische Soldaten und Fronteinheiten der Legion dienen. Sie sind furchtlos und unersch�tterlich loyal gegen�ber dem gottgleichen Willen ihrer Meister. Man sieht sie fast immer mit schweren, beidh�ndigen Waffen. Sie schlagen zwar langsam, aber ziemlich hart zu und sind beweglicher, als sie aussehen.;;;;;;;;;;;x
 EVTDESC_WCCLS_1215_success;When the rift into the Twisting Nether was opened, you spotted the silhouette of an enormous creature on the other side. Its thoughts were only darkness, but you managed to successfully subdue the beast. Following its domination, the horrifying slimy mass of flesh steps through the portal and bows its head to you.;;Als der Spalt in den wirbelnden Nether ge�ffnet wurde, saht Ihr die Silhouette einer gewaltigen Kreatur auf der anderen Seite. Seine Gedanken waren von Dunkelheit erf�llt, aber Ihr konntet die Kreatur unterwerfen. Die schreckliche Masse aus schleimigem Fleisch trat in Eure Dimension und neigte seinen Kopf vor Euch.;;;;;;;;;;;x
 EVTDESC_WCCLS_1215_fail;When the rift into the Twisting Nether was opened, you spotted the silhouette of an enormous creature on the other side. Almost immediately, its tentacles reached through towards you. With a grand sweeping motion, your limbs are shattered, and your body is yanked through the portal you created. The portal closes, and you find yourself alone with the Eldritch creature in total darkness.;;Als der Spalt in den wirbelnden Nether ge�ffnet wurde, saht Ihr die Silhouette einer gewaltigen Kreatur auf der anderen Seite. Seine Tentakel reichten bis zu Euch. Mit einem leichten Ruck brachen sie Euch alle Gliedma�en und zogen die Reste Eures K�rpers in das Portal. Als es sich geschlossen hatte, befandet Ihr Euch allein mit der Kreatur in v�lliger Dunkelheit.;;;;;;;;;;;x
-EVTOPTA_WCCLS_1213;Summon a c'thrax!;;Beschw�rt einen C'Thrax!;;;;;;;;;;;x
-EVTOPTB_WCCLS_1213;Summon a n'raqi!;;Beschw�rt einen N'Raqi!;;;;;;;;;;;x
-EVTOPTC_WCCLS_1213;Summon a zoatroid!;;Beschw�rt einen Zoatroiden!;;;;;;;;;;;x
+EVTOPTA_WCCLS_1213;Summon a C'Thrax!;;Beschw�rt einen C'Thrax!;;;;;;;;;;;x
+EVTOPTB_WCCLS_1213;Summon a N'Raqi!;;Beschw�rt einen N'Raqi!;;;;;;;;;;;x
+EVTOPTC_WCCLS_1213;Summon a Zoatroid!;;Beschw�rt einen Zoatroiden!;;;;;;;;;;;x
```

Comment:
Why they're capitalized?

Reply:
why not? https://wow.gamepedia.com/C%27Thrax
removed capitals from n'raqi and zoatroid, but C'Thrax seems right